strawberry_django_jwt/middleware.py

```diff
@@ -110,8 +111,17 @@ def resolve(self, _next, root, info: GraphQLResolveInfo, *args, **kwargs):
         return _next(root, info, **kwargs)
 
 
+def channels_compat(context: StrawberryChannelsContext) -> None:
```

Comment:
I wouldn't put that in `utils` since it is used only here. This is a function for readability.